<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PingManager test</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>10</y>
     <width>125</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>ConnectState</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbDisconnected">
      <property name="text">
       <string>Disconnected</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbConnecting">
      <property name="text">
       <string>Connecting</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbConnected">
      <property name="text">
       <string>Connected</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbDisconnecting">
      <property name="text">
       <string>Disconnecting</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>10</y>
     <width>132</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>NetworkDetectionManager</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="cbIsOnline">
      <property name="text">
       <string>IsOnline</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edNetworkName"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnSetNetworkName">
      <property name="text">
       <string>Set network name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnUpdateIps">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>217</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Update IPs from the file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
